docs/sources/flow/monitoring/agent-resource-usage.md

```diff
@@ -0,0 +1,78 @@
+---
+aliases:
+  - /docs/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/monitor-infrastructure/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/send-data/agent/flow/monitoring/resource-usage/
+canonical: https://grafana.com/docs/agent/latest/flow/monitoring/resource-usage/
+description: Guidance for expected Agent resource usage
+headless: true
+title: Resource usage
+---
+
+# {{% param "PRODUCT_NAME" %}} resource usage
+
+This page provides guidance for expected resource usage of 
+{{% param "PRODUCT_NAME" %}} for each telemetry type, based on operational 
+experience of some of the {{% param "PRODUCT_NAME" %}} maintainers.
+
+{{% admonition type="note" %}}
+
+The resource usage depends on the workload, hardware and the configuration used.
+The information on this page is a good starting point for most users, but your
+actual usage may be different.
+
+{{% /admonition %}}
+
+## Prometheus metrics
+
+The Prometheus metrics resource usage depends mainly on the number of active
+series that need to be scraped.
+
+As a rule of thumb, **per each 1 million active series** 
+you can expect to use approximately:
+
+* 1.5 CPU cores
+* 11 GiB of memory
+* 1.5 MiB/s of total network bandwidth, send and receive
```

Comment:
@mattdurham @jcreixell - this is based on experimental data from setting scrape interval to 1m. I think it's the best data we have right now until we start doing benchmarks more systematically and I'd recommend we go with this for now. LMK if you have any objections :)